### ISSUE TITLE
OpenRC

### ISSUE THREAD

**Issue by julie-de-ville** [fepitre/qubes-gentoo#32]
I am trying to install the qubes tools on a Pentoo HVM to convert it into a template, and Pentoo uses OpenRC natively. Currently, it is trying to install systemd packages, which are blocking packages like OpenRC. Is there any workaround? I had thought that I would be able to use the qubes packages by writing the necessary service files for them, but I am kinda stuck now.

**Comment by fepitre** [comment]
I don't think there is a workaround as it is built with systemd profile (see https://github.com/QubesOS/qubes-builder-gentoo/blob/main/scripts/distribution.sh#L201). At first, I guess you need to do a separate overlay for that and start with:

- [qubes-libvchan-xen](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-libvchan-xen)
- [qubes-db](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-db)
- [qubes-core-qrexec](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-core-qrexec)
- [qubes-core-agent-linux](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-core-agent-linux)
- [qubes-gui-common](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-gui-common)
- [qubes-gui-agent](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-gui-agent)
- [qubes-utils](https://github.com/fepitre/qubes-gentoo/tree/release4.2/app-emulation/qubes-utils)

You can check what as been done for Alpine Template with OpenRC (see https://github.com/QubesOS/qubes-issues/issues/7323).

**Comment by ayakael** [comment]
Indeed, you can look over in my [qports](https://ayakael.net/forge/qports) repo and see the openrc scripts that I made for the Alpine template.
